xtask/src/bootstrap.rs

```diff
@@ -80,7 +81,7 @@ impl Bootstrap {
             Loader::compute_control_id_table(&CpuHal::new(Blake2bCpuHashSuite::new_suite()));
 
         let contents = format!(
-            include_str!("templates/control_id_rv32im.rs"),
+            include_str!("templates/control_id_rv32im.rs.tmpl"),
```

Comment:
I did this because the new functions will get formatted in a way that breaks their ability to be used as a template. Changing the final extension to `.tmpl` keeps most syntax highlighting (for me) while avoiding having the editor or our tooling try to enforce formatting.